aiida_common_workflows/workflows/relax/cp2k/generator.py

```diff
@@ -310,8 +311,8 @@ def get_builder(
         builder.cp2k.metadata.options['parser_name'] = 'cp2k_advanced_parser'
 
         # Uncomment to change number of CPUs and execution time.
-        # builder.cp2k.metadata.options['resources'][ "num_mpiprocs_per_machine"]  = 12
-        # builder.cp2k.metadata.options['max_wallclock_seconds']  = 3600 * 24
+        #builder.cp2k.metadata.options['resources']['num_mpiprocs_per_machine'] = 16
+        #builder.cp2k.metadata.options['max_wallclock_seconds'] = 3600 * 72
```

Comment:
I would remove these lines. These can be set directly in the `engines` argument, _and_ they can be set through the CLI as well. So do we need to have this hardcoded and then commented out in the generator?

Reply:
I used this for testing, but if it is possible to set those things from the CLI, sure - we can remove them.